Code/GraphMol/molopstest.cpp

```diff
@@ -7735,6 +7735,7 @@ void testGithub1614() {
       // nm.debugMol(std::cerr);
       bool force = true, cleanIt = true;
       MolOps::assignStereochemistry(nm, cleanIt, force);
+      // nm.debugMol(std::cerr);
```

Comment:
Would you mind cleaning up this commented debug print, and the one above?

Reply:
Will do